misk-action-scopes/src/test/kotlin/misk/scope/ActionScopedTest.kt

```diff
@@ -169,7 +170,7 @@ internal class ActionScopedTest {
 
     )
     scope.enter(seedData).use { actionScope ->
-      runBlocking(actionScope.asContextElement()) {
+      runBlocking((actionScope as ActionScope).asContextElement()) {
```

Comment:
I guess we can only determine if `asContextElement` needs to be part of the interface (and then this cast can be removed) when we implement the armeria side. I think it's ok to go with that version for now.

Reply:
Yes, we have decided to not add "asContextElement" in the generic interface for now. I will add it if required by the downstream dependencies.